### PR TITLE
SNOW-1333462 update libsfc 1.0.11 built with gcc8

### PR DESCRIPTION
**Pull request by SimbaGithub**
Update libsfclient 1.0.11 built with gcc8

### CODE REVIEW

File: libsnowflakeclient/deps-build/darwin/aws/include/aws/auth/auth.h [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#ifndef AWS_AUTH_AUTH_H
```
Comment (sfc-gh-dprzybysz) [inline libsnowflakeclient/deps-build/darwin/aws/include/aws/auth/auth.h:1]: Do we need header files of the libsnowflakeclient dependencies? can we drop them?

**Review by sfc-gh-dprzybysz — COMMENTED** [review]
Please update PR name with issue id and version 1.0.11 of libsnowflakeclient

**Comment by sfc-gh-dprzybysz** [comment]
Please check also include files of libsnowflakeclient - some files seems to be missing

File: libsnowflakeclient/deps-build/darwin/aws/include/aws/auth/auth.h (hunk 1 above)
Comment (SimbaGithub) [reply libsnowflakeclient/deps-build/darwin/aws/include/aws/auth/auth.h]: done

**Review by sfc-gh-dprzybysz — APPROVED** [review]

**Pull request merged**